employees/templates/employees/author_report_list.html

```diff
@@ -18,11 +18,17 @@ <h1>{{ object.email }} {{ title_date }}{{ UI_text.PAGE_TITLE.value }}</h1>
 <a href="{% url 'custom-users-list' %}" class="btn btn-primary hidden-print">
     {{ UI_text.RETURN_BUTTON_MESSAGE.value }}
     {% with date=path|extract_year_and_month_from_url %}
```

Comment:
It just got to me, that you could simply pass `month` and `year` to context in here https://github.com/Code-Poets/sheetstorm/blob/master/employees/views.py#L103 and then remove `{% with date=path|extract_year_and_month_from_url %}`. Please do so in this pull request (here and in other places, and remove `extract_year_and_month_from_url` template tag then, as it won't bee needed anymore) as following comment will depend on it.

Reply:
I think I have done, but I didn't delete it from templetastag, should I ?

Reply:
If it is not used, then yes. Also tests for it.

Reply:
Done